src/components/pages/Styleguide/StyleguideElements/StyleguideCheckboxes.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+
+import { CheckboxInput } from 'components/layout/inputs/CheckboxInput/CheckboxInput'
+
+const StyleguideCheckboxes = () => {
+  const [checked, setChecked] = useState(false)
+
+  const checkboxeSnippet = String.raw`
```

Comment:
```suggestion
  const checkboxeSnippet = `
```
Ne ne fonctionne pas juste comme ça ?

Reply:
J'ai repris l'existant dans les autre style guide, mais effectivement, le String.raw n'est pas utile.